hack/check_golden_images.sh

```diff
@@ -5,6 +5,12 @@ set -ex
 IMAGES_NS=${IMAGES_NS:-kubevirt-os-images}
 
 if [[ $(${KUBECTL_BINARY} get ssp -n ${INSTALLED_NAMESPACE}) ]]; then
+  [[ $(${KUBECTL_BINARY} get imageStream centos8  -n ${IMAGES_NS} --no-headers | wc -l) -eq 0 ]]
```

Comment:
if you use `get` for a specific object by name, why counting the output lines and comparing to 0?
whether it's exist, or not.
you can simplify with
```
[[ ! $(${KUBECTL_BINARY} get imageStream centos8 -n ${IMAGES_NS} ]]
```